sdk/servicebus/service-bus/src/serviceBusMessage.ts

```diff
@@ -620,11 +620,27 @@ export function isServiceBusMessage(possible: unknown): possible is ServiceBusMe
   return isObjectWithProperties(possible, ["body"]);
 }
 
+export let _featureAmqpBodyTypeEnabled = false;
+
+/**
+ * Enables preview AMQP body type support.
+ * @internal
+ */
+export function setFeatureAmqpBodyTypeEnabled(enable: boolean): boolean {
+  const previousState = _featureAmqpBodyTypeEnabled;
+  _featureAmqpBodyTypeEnabled = enable;
+  return previousState;
```

Comment:
The pattern of a "setter" returning previous value is new to me.. Wouldnt this be confusing?

Reply:
Given that this is mainly to test a feature that is otherwise disabled, why not use two separate methods `enableAMQPBodyTypeFeaturForTesting()` and `resetAMQPBodyTypeFeatureAfterTesting()` to make it absolutely clear that these are temporary measures?

Reply:
I could definitely do that. 

This particular technique is relatively common with atomic style operations but...it's a little weird. :) 

I'll do the separate and rename real quick.